account_move_line_product/views/account_move_line_view.xml

```diff
@@ -46,16 +32,4 @@
         </field>
     </record>
 
-    <record id="view_move_form" model="ir.ui.view">
-        <field name="name">account.move.form</field>
-        <field name="model">account.move</field>
-        <field name="inherit_id" ref="account.view_move_form"/>
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='line_ids']/tree//field[@name='partner_id']" position="after">
-                <field name="product_id"/>
-                <field name="quantity"/>
-            </xpath>
-        </field>
-    </record>
```

Comment:
Why are the view removed completely? product and qty are always displayed in v13?

Reply:
Sorry, I misunderstand. Edited.